Mentor session details

cdpaiva opened the pull request:
This PR adds:

1. Makes sure that the `mentorContext` is properly populated in the pages that it is required (navigates back to the mentor homepage otherwise);
2. Updates the URL to get the cohorts of a mentor to match the new API endpoint;
3. Adds the details of a session: mentors will see the details of a session, the confirmed participants and the discussion. I ended up creating a different interface, let me know what you think! For reference:

![image](https://github.com/Code-the-Dream-School/dd-prac-team2-front/assets/55894222/4b3c099b-f15c-4abe-a5e9-a4eecc870c83)

cdpaiva committed:
Updates endpoint

cdpaiva committed:
Redirects to home page if context is empty

cdpaiva committed:
Adds session details for mentors

cdpaiva committed:
Finishes first version of mentor session details

secch97 reviewed (approved):
Looks pretty great Carlos! Amazing! I would only suggest to change the colors to the app's palette and adding a border radius to the main container of your UI, comment item containers and adding a margin between each comment. Let me know what do you think about this.

cdpaiva committed:
Interface updates

cdpaiva commented:
Ok! I've just added the border radius and the margins as you suggested. I also changed the 'Add new Comment' button, to match the style in the `AppButton` component.
